lib/board.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/utils'
+
+class Board
+  include Utils
+
+  attr_accessor :display_board
```

Comment:
- [ ] [OPTIONAL] Only use `attr_accessors` if you really need to. Using `attire_accessors` means that the property can be set to anything from outside the class. ( [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).

Reply:
Update attr_accessor to attr_reader in board class